tests/infra/remote.py

```diff
@@ -584,6 +585,12 @@ def __init__(
         # to reference the destination file locally in the target workspace.
         enclave_path = os.path.join(".", os.path.basename(lib_path))
 
+        election_timeout_arg = (
```

Comment:
Should we explicitly construct and pass the heartbeat/status arguments to `cchost` here as well (instead of relying on the default values in the executable)?

Reply:
I thought about it a bit, I would say to not expose them until we actually need to configure them from the test infra.